angular/projects/spark-angular/src/lib/components/sprk-toggle/sprk-toggle.component.spec.ts

```diff
@@ -44,37 +40,80 @@ describe('SprkToggleComponent', () => {
     const str = 'One';
     component.analyticsString = str;
     fixture.detectChanges();
-    expect(element.querySelector('button').getAttribute('data-analytics')).toEqual(
-      str
-    );
+    expect(
+      element.querySelector('button').getAttribute('data-analytics'),
+    ).toEqual(str);
   });
 
   it('should add icon classes to icon when toggle is opened', () => {
-    component.title = 'placeholder';
+    component.triggerText = 'placeholder';
     element.querySelector('button').click();
     fixture.detectChanges();
     expect(
-      element.querySelector('button .sprk-c-Icon').classList.toString()
+      element.querySelector('button .sprk-c-Icon').classList.toString(),
     ).toEqual(
-      'sprk-c-Icon sprk-c-Icon--xl sprk-u-mrs sprk-c-Icon--toggle sprk-c-Icon--open'
+      'sprk-c-Icon sprk-c-Icon--xl sprk-u-mrs sprk-c-Icon--toggle sprk-c-Icon--open',
     );
   });
 
   it('should add icon classes to icon when the toggle is opened and then closed', () => {
-    component.title = 'placeholder';
+    component.triggerText = 'placeholder';
     element.querySelector('button').click();
     element.querySelector('button').click();
     fixture.detectChanges();
     expect(
-      element.querySelector('button .sprk-c-Icon').classList.toString()
+      element.querySelector('button .sprk-c-Icon').classList.toString(),
     ).toEqual('sprk-c-Icon sprk-c-Icon--xl sprk-u-mrs sprk-c-Icon--toggle');
   });
 
+  it('should render open if isOpen is set to true', () => {
```

Comment:
This is all semantics, but I wonder if this test is really "should load correct classes if isOpen is set to true"